packages/interactivity/src/directives.tsx

```diff
@@ -50,6 +50,39 @@ function deepClone< T >( source: T ): T {
 	return source;
 }
 
+function wrapEventAsync( event: Event ) {
```

Comment:
Add jsdoc for this?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/68097/commits/2dd6eabe1a7d7ade2f3ff1834dd7232551498e72